agot-bg-game-server/src/common/ingame-game-state/game-data-structure/GameLog.ts

```diff
@@ -150,6 +150,8 @@ interface RaidDone {
     raiderRegion: string;
     raidedRegion: string | null;
     orderRaided: number | null;
+    raiderGainedPowerToken: boolean | null;
+    raidedHouseLostPowerToken: boolean | null;
```

Comment:
AFAIK when deserializing old logs without that values ts initializes them with `null`. So I guess there is no migration needed. At least @Longwelwind merged my march log PR already where the same technique is applied so I guess it's ok.

Reply:
Yeah, it will work correctly since the values will be `undefined`.

Logs from all games will not be super consistent, but it's a small loss, I think.

Reply:
Not sure if typescript overrides the javascript behavior afaik it doesn't, but when accessing any non-initialized values (declared but non-initialized variables or attributes of an object) it has a value of `undefined` instead of raising an error. `undefined` behaves similarly to `null` but isn't quite the same:
```javascript
let x = {};
console.log(x.a);
// undefined
x.a = null;
console.log(x.a)
// null
```

Reply:
Yeah, but any comparison with `==` will have the same behavior than having `null`.